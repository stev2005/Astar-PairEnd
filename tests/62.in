ACGAACAAAACTGTAGGTCTTCAGATGGCTTCAGGATGAAGCTAATGTGAGGCTAGGTATGTCGCTTCCTCTCAACTGAGGTAAGAGATGACCCATATACGCATCACATCGCAGCGTTTGCTATAAGATGCCTCCAGGACTTGACAGATCTCATAATGAGACTTTATCGTGGGAGAAAAAGTCGGCTCTTTTCTGATCCGATATTGTAGTAATCTCGGTCCGAGCGGCTTTTGTGGCCGTCGAATTCCCAGGTGCAATCTTAG
15
TCTAGCATTATGTCTACCGATTCTTGGGTCTGGTTAATAATTGGAGGGTAGTATGTGCCCGATATGAACAAAAGGAAAGACCTCACAGGCTACTACCCCTCCTCTGGAGAATCTAGAAATCATTTAAACCTGGTACCGCTGCTAATGCAG
CGTACTAGACCTTATCCCGCT
ATATCCTACCGCCAAGGCTTATATACAGGTGGTTGAACAGGTTATTGCACAAAATAGTTATCTGACGCGTTCGGCCCTGGAGGCTGCCCACAGAGGCCTAAGCGAGTGCCACTGGGAATGACGGCCCTGATGTAATGTCTAC
TATGATCGTGCGAGTTATGGGGGGACTTCCTACGGCCAACGCTGTGCTCTCACT
CACACGACTGACAACCGGACCGGATGCTTACCATCCCCGCAGGAGTCCCCGGTAGGGTGGTTTTGAAAGGCGCTGTGTATAGCTTTCCGTTCTGACTACCGGTTCCTTATGAGTTCGAGAAGCTACATCGAAATTGAGCAGAAATTCGATGACATGCTCTCGTGCGAGCCGTCTTCGATGTCGCCCTTCAGGGCAATAGAGATTTGAGGTAGCTTGAGCCGTACAACACTATTAGTGTCCGCTT
GGCATGGTGTTTATATATTACGGATTTAGGCAAGAAATTTGGGTGGGGCCGGTATGAGTGAATCGTCGGACCGTAACGGGAACTCAGCGCTGCGTTGACAATCTGCTAAGGAGTTTTCACGAATGTGAAACATGTCATCACAATTTTTCT
TTATGACGACTAAGAGGTTGATGTTCGGGGGGCTCTTGGTACTATTGCGGTGCCGATAGCGCTAAATTTCGTTC
GCGATATGCCAGAAACTCCTCCGAAAGTTATCCCCACCGGGGAGTCTGGACAGTAGTTGGTCAAGCATGTCACGGATGGCGTCATGTGCCCAGCAAGATCTACCGACCGATATACGGACTCGTTTTTCAGGTGAACAGGGGCGCGTAATCAATTTGTGTTAGGTGTAGCGATAGAG
AGCCGGAGCGCAGAAAGTGCTAGGACGGGAAGGGACAATCTACCAGCGCTACTGTTACCGCGATAAAACTCATTCTCGGGCGTAAGTATAGAGTAGTAGAAAATTCTATCGATGATATAAAGAGCACCCTCCTGAGTTGTACTTTATTTTTTCAGTATACGCGGTTATGTGTACGTCCGCAGAGGGCGCCAAGGGGCGCA
TTCACAGTTTTCTCTGGCTTCTGACACACCCAATAGTGCGCATCGGTAAGACGGCTGTTTACTCATTTGAGTACAGATTAGTCACTTTGTGGACTAAGTGGCGTTGCTGATATAAATTTCGCAGGTGTGGCATTTGGCCACAACCACATCGGGGACCTATCCAACTTTAATGCTTGTAGGCAAAGCCAACTCGTCTT
GTCCCGAAATCG
GTGGCTTCTACATAGGTGCTTGACTCCGGAAATTTAGGGGTTGGTAATTCGGAGATTCCCCGGACCCTTTCGGAACACATGGGGCGCATTGCCACGCGCACTTTTTCTACTTACCCTTCGGTTTTCCATGCCCAAATCAATTTTAAATTCCCCCAAGCCCAGGCGTGCAGCTCAGCCGAAACGCGGCAAGGAAACGTTAACCTGGGTTATTACCCTATAACGAGGCAGCAGGGCCATATAAGACA
AACAAGGAACCCTGC
CTGGTGGGGGCGTCAACCACATCAA
CTCTAGGGAGAGAGCCCAGCATTCATACTAACTCATTGCTAGCCAGGCTCTTAGACCAGAAGCTTCGGAACAGTCGCAAACTACCAGGCCGCTACTCTCCTTGCGTATTAAATAGAAAGCCATCAAAGTCTTACAGCAGAAGCTGTTGTCACGTGGTGAGTACACGGGGATTGAGATGTGTTTCGCACTATTACTGTCA
