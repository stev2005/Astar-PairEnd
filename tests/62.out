70
7
68
23
117
71
33
87
94
91
4
126
5
10
94
